pkg/types/interfacetests/codec_interface_fuzz_tests.go

```diff
@@ -32,6 +32,14 @@ func RunCodecInterfaceFuzzTests(f *testing.F, tester CodecInterfaceTester) {
 				fb[index] = fixedBytes[index]
 			}
 
+			// put a limit on the size of strings because some encodings will need to know ahead of time
+			if len(s) > 100 {
+				s = s[:100]
+			}
+			if len(differentField) > 100 {
+				differentField = differentField[:100]
+			}
```

Comment:
Would it make sense to just `t.Skip()` these cases instead? My thought was that rejecting the input instead of just mutating it would inform the fuzz runtime not to explore that kind of input.

Reply:
I think I can just revert this.  I was thinking about chains, but this is just for the codec.  The codec test doesn't need to encode the exact same way as the chain test :).